core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -90,6 +94,10 @@ protected final O executeDelegate(ClientRequestContext ctx, I req) throws Except
         }
     }
 
+    protected static Logger logger() {
+        return logger;
```

Comment:
I'm not sure we really need this. A subclass should use its own `Logger`? e.g. `RetryingHttpClient` should use the `Logger` with the name `RetryingHttpClient`, not `RetryingClient`.